src/components/TodoList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { TodoItem } from './TodoItem';
+import { Todo, Todos } from '../types/todoTypes';
+
+interface Props {
+  items: Todos;
+  selectedFilter: string;
+}
+
+function FilteredItems(items: Todo[], selectedFilter: string) {
```

Comment:
```suggestion
function filteredItems(items: Todo[], selectedFilter: Filter) {
```
we name function in camelCase